target/linux/ipq40xx/image/Makefile

```diff
@@ -220,6 +220,22 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_eap2200
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := EnGenius
+	DEVICE_MODEL := EAP2200
+	BOARD_NAME := eap2200
+	DEVICE_DTS := qcom-ipq4019-eap2200
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_PACKAGES := ath10k-firmware-qca9888
+	IMAGES := nand-factory.bin nand-factory.ubi nand-sysupgrade.bin
+	IMAGE/nand-factory.bin := append-ubi | qsdk-ipq-factory-nand
```

Comment:
Is something using the "nand-factory.bin" and "nand-factory.ubi" files? As the installation method in the commit message doesn't mention these files.